video/flic_decoder.cpp

```diff
@@ -363,7 +361,7 @@ void FlicDecoder::FlicVideoTrack::unpackPalette(uint8 *data) {
 	if (0 == READ_LE_UINT16(data)) { //special case
 		data += 2;
 		for (int i = 0; i < 256; ++i) {
-			memcpy(_palette + i * 3, data + i * 3, 3);
+			_palette.set(data + i * 3, i, 1);
```

Comment:
Thinking this could be _palette.set(i, data[i *3], data[i *3 + 1], data[i *3 + 2])